### ISSUE TITLE
JCR Properties- adding cq:designPath to jcr:content(PageContent) node

### ISSUE THREAD

**Issue by AliShirazi** [adobe/aem-eclipse-developer-tools#62]
Hey guys,

I am getting a weird error when adding the cq:designPath to the jcr:content PageContent node. It says "The prefix "cq" for attribute "cq:designPath" associated with an element type "jcr:content" is not bound. Thinking the property type was the issue, I then tried to manually set it to String but no luck. It just stays empty. I also made sure the property was listed before jcr:primaryType. Even though the property still shows in the .content.xml on my local system, it does not auto-update in crxde lite unless I force push it to the AEM server or edit the node in crxde lite.

If I add the property in crxde lite and then "Import from Server" using the AEM plugin, there are no longer any errors about the cq:designPath property and the String type is automatically assigned. If I then remove and readd the property, I get the error again. 

![screen shot 2016-02-03 at 3 18 51 pm](https://cloud.githubusercontent.com/assets/5385782/12797685/c3dc2daa-ca8a-11e5-8fe8-0b46ce0d5f78.png)

**Comment by rombert** [comment]
Thanks for the report. How do you add the property, with the XML editor or via the JCR properties view?

**Comment by AliShirazi** [comment]
Hey Robert,

Not a problem, thanks for the quick reply! I tried to add it with both the XML editor and the JCR properties view. 

Eclipse Version: Mars.1 Release (4.5.1)

I'm also using the latest version of the AEM plugin

**Comment by AliShirazi** [comment]
Robert,

It looks like this is actually a namespace issue, similar to https://github.com/Adobe-Marketing-Cloud/aem-eclipse-developer-tools/issues/50. I noticed the .context.xml file imported from CRXDE lite had xmlns:cq="http://www.day.com/jcr/cq/1.0".  I mentioned above about the issue with adding/re-adding, but I can't reproduce it after restarting eclipse and having the namespace there.

I guess CRXDE lite inputs them automatically in the .content xml file so unfortunately I did not think to double-check
